lambdas/tests/farsetlabs/handlers/producer.test.js

```diff
@@ -6,16 +6,18 @@ const { uploadTo } = require(`${prefix}/node_modules/@muxer/lambda-utils`);
 const producer = require(`${prefix}/handlers/producer`);
 
 const apicall =
-  "https://www.googleapis.com/calendar/v3/calendars/farsetlabs.org.uk_srmqnkn373auq51u00s2nijrq8%40group.calendar.google.com/events?maxResults=2500&singleEvents=true&orderBy=startTime&timeMin=2019-11-06T10:30:00.000Z&timeMax=2020-11-06T10:30:00.000Z&key=google-calendar-token-abc123";
+  "https://www.googleapis.com/calendar/v3/calendars/farsetlabs.org.uk_srmqnkn373auq51u00s2nijrq8%40group.calendar.google.com/events?maxResults=2500&singleEvents=true&orderBy=startTime&timeMin=2018-11-06T10:30:00.000Z&timeMax=2019-11-06T10:30:00.000Z&key=google-calendar-token-abc123";
 
 const event = null;
 const context = null;
 
 describe("Farsetlabs Producer", function() {
   beforeEach(function() {
-    const frozenDate = new Date("2018-11-06T10:30:00");
+    const NativeDate = Date;
+    global.Date = jest.fn(
+      input => new NativeDate(input || "2018-11-06T10:30:00")
+    );
```

Comment:
There was an issue with the previous stubbing of this - if we just return an existing date object, that can be mutated by the logic calling it. So instead, this now creates a new date object on each call, or defaults calls without a specific time to 6th November 2018.